Ofqual.Recognition.Citizen.API.Core/Models/DTO/TaskStatusRawDto.cs

```diff
@@ -0,0 +1,14 @@
+using Ofqual.Recognition.Citizen.API.Core.Enums;
+
+namespace Ofqual.Recognition.Citizen.API.Core.Models;
+
+public class TaskStatusRawDto
```

Comment:
I would like to see some documentation at least which defines the difference between what TaskStatusDto is and what TaskStatusRawDto is. The naming for me is also incorrect - this isn't really a DTO as DTOs are used purely for transferring to and from the client; it might be better to name this something like TaskStatusSection as this appears to contain both the TaskStatus information _and_ the Section information joined together, and is used to pull the data from the database. This is also where the ISection and ITaskStatus interfaces will become useful as that will make it clearer that this model includes aspects of both bits

Reply:
Resolved in latest changes